Interfaces/debugger.h

```diff
@@ -228,7 +228,7 @@ class DebuggerInformation : public SerializedObject
         arch.Write(wxT("useRelativeFilePaths"), useRelativeFilePaths);
         arch.Write(wxT("maxCallStackFrames"), maxCallStackFrames);
         arch.Write(wxT("catchThrow"), catchThrow);
-        arch.Write(wxT("showTooltips"), showTooltipsOnlyWithControlKeyIsDown);

```

Comment:
I am pretty sure that I modified this file last week, can you please do a rebase and re-submit the PR?

Reply:
Rebased on commit 3736b98.